doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1541,19 +1541,25 @@ whose general syntax is
    :name: =>
 
 .. prodn::
-   i_item ::= @i_pattern %| @s_item %| @clear_switch %| {? %{%} } /@term
+   i_item ::= @i_pattern %| @s_item %| @clear_switch %| @i_view %| @i_block
 
 .. prodn::
-   s_item ::= /= %| // %| //=
+   i_view ::= {? %{%} } /@term %| /ltac:( @tactic )
 
 .. prodn::
-   i_pattern ::= @ident %| _ %| ? %| * %| {? @occ_switch } -> %| {? @occ_switch }<- %| [ {?| @i_item } ] %| - %| [: {+ @ident } ]
+   s_item ::= /= %| // %| //= %| /@num= %| /@num/ %| /@num/@num=
 
-The ``=>`` tactical first executes tactic, then the :token:`i_item` s,
+.. prodn::
+   i_block ::= [^ @ident ] %| [^~ @ident ] %| (^ @ident ) %| (^~ @ident )
+
+.. prodn::
+   i_pattern ::= @ident %| > @ident %| _ %| ? %| * %| + %| {? @occ_switch } -> %| {? @occ_switch }<- %| [ {?| @i_item } ] %| ( {?| @i_item } ) %| - %| [: {+ @ident } ]
+
+The ``=>`` tactical first executes :token:`tactic`, then the :token:`i_item` s,
```

Comment:
Now that I've made you change this, I only realize that `tactic` is not a defined grammar token. Never mind in the grammar of Ltac there is the notion of `atomic_tactic`. We should rename this `tactic`. Is this what's intended here or are tactic expressions also accepted?

Reply:
it is correct, you can write `(tac; tac) => ipats`.